### DISCUSSION TITLE
Branching strategia

### DISCUSSION THREAD

**Discussion by nnorbi3** [SzFMV2021-Osz/AutomatedCar-A#12] [General]
Sziasztok!
A csapat kozotti conflict-ok elkerulese erdekeben, a projekt elkeszitesehez milyen branching strategiat hasznalunk? Trunk based/gitflow?

**Comment by pintergreg** [comment]
[GitHubFlow](https://guides.github.com/introduction/flow/), ez elég jól működött. GitFlow-hoz képest nincs drasztikus eltérés.
Összehasonlítás a többivel és a részletesebb cikk ezekről elérhető [a jegyzeten keresztül](https://szfmv2021-osz.github.io/handout/github.html?highlight=githubfl#branching-modell).
